requirements.txt

```diff
@@ -3,7 +3,6 @@ psycopg2==2.5.2
 py3k-bcrypt==0.3
 stripe==1.11.0
 pytz==2014.4
-pep8==1.6.2
```

Comment:
Since we have `requirements_test.txt` now, I figured that would be a better place to require `pep8`.